engine/crates/grafbase-engine/src/schema.rs

```diff
@@ -723,7 +727,7 @@ async fn process_deferred_workload(
         label: workload.label,
         data,
         path: workload.path,
-        has_next: true, // TODO: Need to handle has_next properly...
+        has_next: true, // We hardcode this to true here, the function calling us should override
```

Comment:
IMHO should be `false` by default then, feels like supposing there is another workload when it's not even aware of it. but super tiny nitpick obviously.^^